src/GitHub.VisualStudio/Views/GitHubPane/PullRequestFilesView.xaml

```diff
@@ -60,27 +66,8 @@
                             Tag="{Binding DataContext, ElementName=root}"
                             KeyboardNavigation.DirectionalNavigation="None">
 
-                    <!-- 
-                      We need to change the color of the file icon when the file is deleted, but applying the style
-                      to OcticonImage directly borks the designer (see #1410). Work around this by applying the color
-                      to a parent control and let the foreground be inherited by the icon.
-                    -->
```

Comment:
I noticed this used to gray out the icon if the item was deleted. This can't really be done with arbitrary icons. I tried replacing the color change with reduced opacity, but it didn't look very good in my opinion. And the built-in team explorer view doesn't do anything like that either (it's only the text that is grayed out). Is it OK to remove this?

Reply:
Yeah that's fine with me!